src/components/LightBox.vue

```diff
@@ -40,6 +40,7 @@
               v-else
               ref="video"
               controls
+              :key="media[select].sources[0].src"
```

Comment:
Sure! I use to put the key property as the first of all, because key property is a must-have for a component to render. So, the key needs priority over the others.

Reply:
Well, I follow the convention that places `key` as the last property. 

I think the convention should be up to the owner @pexea12

Reply:
I think it doesn't make any difference

Reply:
So for me as well. Just a convention which if you follow the whole project the `key` property is mostly on the top, as I did.